extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -138,7 +139,20 @@ export class AmpAdXOriginIframeHandler {
         this.adResponsePromise_,
         'timeout waiting for ad response').catch(e => {
           this.noContent_();
-          user().warn('AMP-AD', e);
+          getAdCid(this.baseInstance_).then(cid => {
+            const tagType = this.baseInstance_.element.getAttribute('type');
+            const TAG = `AMP-AD-${tagType}`;
```

Comment:
This should probably be moved to class-level and used uniformly in all messages.  ( @keithwrightbos just did that for amp-a4a.js, but I don't think he made that change here.  Might as well be consistent.)